internal/translation/auditing/conversion.go

```diff
@@ -0,0 +1,56 @@
+package auditing
```

Comment:
```suggestion
package audit
```
shorter?

Reply:
As `audit.AuditLog` for the interface and `audit.AuditLogService` for the struct?

BTW I usually think the other way around, `...service` for the abstract interface while the other seems to be more appropriate for the struct. Will think about it, might be a remnant from my old Java days.

Reply:
> As audit.AuditLog for the interface and audit.AuditLogService for the struct?

no, `audit.AuditLogService` for the interface and `audit.AtlasAuditLogService` for the struct.